ui/packages/consul-ui/app/helpers/hcp-authentication-link.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Helper from '@ember/component/helper';
+import { inject as service } from '@ember/service';
+
+/**
+ * A resourceId Looks like:
+ * organization/b4432207-bb9c-438e-a160-b98923efa979/project/4b09958c-fa91-43ab-8029-eb28d8cee9d4/hashicorp.consul.global-network-manager.cluster/test-from-api
+ * organization/${organizationId}/project/${projectId}/hashicorp.consul.global-network-manager.cluster/${clusterName}
+ *
+ * A HCP URL looks like:
+ * https://portal.cloud.hashicorp.com/sign-in?cluster_name=test-from-api&redirect_to=link-consul-cluster&cluster_version=1.18.0&cluster_access_mode=CONSUL_ACCESS_LEVEL_GLOBAL_READ_WRITE&redirect_url=localhost:8500/services
+ */
+export const HCP_PREFIX =
+  'https://portal.cloud.hashicorp.com/sign-in?redirect_to=link-consul-cluster';
```

Comment:
I think this link should be different - we should put the link we want to actually go to in - the redirect logic will redirect us.

https://github.com/hashicorp/cloud-ui/pull/8688

```suggestion
 * https://portal.cloud.hashicorp.com/services/consul/clusters/self-managed/link-existing?cluster_name=test-from-api&cluster_version=1.18.0&cluster_access_mode=CONSUL_ACCESS_LEVEL_GLOBAL_READ_WRITE&redirect_url=localhost:8500/services
 */
export const HCP_PREFIX =
  'https://portal.cloud.hashicorp.com/services/consul/clusters/self-managed/link-existing';
```

Reply:
that was my guess to. so we don't need the `redirect_to=link-consul-cluster` to be a part of link. 
thank you for checking it!